web-api/openapi/front_office.yaml

```
@@ -57,9 +57,12 @@ paths:
         - search
       summary: Personalize documents for the user
       description: |-
-        Get a list of documents personalized for the given `user_id`.
-        Each document contains the id and the score, where a higher value means that the document matches the preferences of the user better. Scores can be compared only with other scores that belong to the same request; comparing scores of documents that have been obtained through different requests can lead to unexpected results.
-        The documents also contain their properties if this is requested and the properties are not empty.
+        Get a list of snippets personalized for the given `user_id`.
+
+        Each snippet has an id and contains the id of the document it is from as well as a score. A higher score means that the document matches the preferences of the user better.
```

:art: maybe like this
```suggestion
        Each snippet has an id and contains the id of the document it originates from as well as a score. A higher score means that the document matches the preferences of the user better.
```